server/schema/type.js

```diff
@@ -0,0 +1,86 @@
+const pageInfoType = `
+  type PageInfo {
+    currentPage: Int!
+    totalPages: Int!
+    hasNextPage: Boolean!
+    hasPreviousPage: Boolean!
+    totalItems: Int!
+  }
+`;
+
+const bookType = `
+    type Book {
+      id: ID!
+      title: String!
+      description: String
+      published_date: String
+      author: Author
+      average_rating: Float
+      review_count: Int
+    }
+`;
+
+const bookListType = `
+  type PaginatedBooks {
+    books: [Book!]!
+    pageInfo: PageInfo!
+  }
+`;
+
+const bookListFilterType = `
+  input BookFilter {
+    title: String
+    authorName: String
+    publishedAfter: String
+    publishedBefore: String
+  }
+`;
+
+const authorType = `
+    type Author {
+      id: ID!
+      name: String!
+      biography: String
+      born_date: String
+      books: [Book]
+    }
+`;
+
+const authorListType = `
+  type PaginatedAuthors {
+    authors: [Author!]!
+    pageInfo: PageInfo!
+  }
+`;
+
+const authorListFilterType = `
+  input AuthorFilter {
+    name: String
+    bornAfter: String
+    bornBefore: String
+  }
+`;
+
+const reviewType = `
+    type Review {
+      id: ID!
+      book_id: ID!
+      user: String!
+      rating: Int!
+      comment: String
+      created_at: String!
+    }
+`;
+
+const type = `
```

Comment:
Any better way to manage this, in large project this might become more painful. 
Also any use of having each type as constant seperately? 
Also How can we split this to make it wor with larger projects?